crates/bevy_ecs/src/world/mod.rs

```diff
@@ -73,9 +73,8 @@ use unsafe_world_cell::{UnsafeEntityCell, UnsafeWorldCell};
 /// Stores and exposes operations on [entities](Entity), [components](Component), resources,
 /// and their associated metadata.
 ///
-/// Each [`Entity`] has a set of components. Each component can have up to one instance of each
-/// component type. Entity components can be created, updated, removed, and queried using a given
-/// [`World`].
+/// Each [`Entity`] can have up to one instance of each type of component. Entity
+/// components can be created, updated, removed, and queried using a given [`World`].
```

Comment:
Perhaps something like
> Each [`Entity`] has a set of components, but only one instance of each type of component.
so that it's clear that multiple types of components can exist, but only one of each type.

```suggestion
/// Each [`Entity`] has a set of components, but only one instance of each type of
/// component. Entity components can be created, updated, removed, and queried using
/// a given [`World`].
```